services/importer/lib/importer/gpx_splitter.rb

```diff
@@ -20,7 +20,7 @@ def self.support?(source_file)
       def initialize(source_file, temporary_directory, ogr2ogr_config = nil)
         @source_file = source_file
         @temporary_directory = temporary_directory
-        @ogr2ogr_binary = @ogr2ogr_config.nil? ? DEFAULT_OGR2OGR_BINARY : ogr2ogr_config['binary']
+        @ogr2ogr_binary = ogr2ogr_config.nil? ? DEFAULT_OGR2OGR_BINARY : ogr2ogr_config['binary']
```

Comment:
So you never use the default one?

This is how it works in CARTO prod right now, would that work?

What about:
```ruby
        @ogr2ogr_binary = if ogr2ogr_config && ogr2ogr_config['binary'].present?
                            `#{ogr2ogr_config['binary']}`.strip
                          else
                            DEFAULT_OGR2OGR_BINARY
                          end
```

Reply:
That code would work as well.

We would use the default if a configuration is not passed in.  However the code (prior to the change) would always use the default since @ogr2ogr_config doesn't exist.

Reply:
I'd just use upstream code for less maintenance! :)